torch/csrc/jit/python/pybind_utils.h

```diff
@@ -927,6 +928,20 @@ inline py::object toPyObject(IValue ivalue) {
     return py::cast(ivalue.toCapsule());
   } else if (ivalue.isFuture()) {
     return py::cast(std::make_shared<PythonFutureWrapper>(ivalue.toFuture()));
+  } else if (ivalue.isEnum()) {
+    auto enum_holder = ivalue.toEnumHolder();
+    auto qualified_class_name = enum_holder->qualifiedClassName();
+
+    auto py_class = py::module::import("torch.jit._state")
+                        .attr("_get_script_class")(qualified_class_name);
+    if (py_class.is_none()) {
+      std::stringstream err;
+      err << "Unknown reference to ScriptClass ";
+      err << qualified_class_name;
+      err << ". (Did you forget to import it?)";
+      throw std::runtime_error(err.str());
```

Comment:
Consider refactoring this, it looks similar to the error message for class types.